modules/sbo-methods-of-exposing-binding-data.adoc

```diff
@@ -169,7 +183,7 @@ metadata:
     ...
 ----
 
-The previous example places the name of the config map in the `metadata.annotations.dbconfig` custom field and specifies to expose a single key from the config map.
+The previous example places the name of the config map in the `{.metadata.name}-config` template that resolves to `hippo-config` eventually. The template can contain multiple JSONPath.
```

Comment:
My previous comment applies to these sentences, too.

Reply:
+1